backend/package.json

```diff
@@ -17,6 +17,18 @@
     "express-oauth2-jwt-bearer": "^1.3.0",
     "node": "^19.6.0",
     "nodemon": "^2.0.20",
-    "pg": "^8.8.0"
+    "pg": "^8.8.0",
+    
```

Comment:
this seems to be a simple formatting thing. Try to get Prettier running on your end and it will do all of that for you. Just so that we keep consistent syting